MovieJournalMVC/Model/Store.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  Store.swift
+//  MovieJournalMVC
+//
+//  Created by Lauren Small on 3/12/19.
+//  Copyright © 2019 Lauren Small. All rights reserved.
+//
+
+import Foundation
+
+class Store {
+    static let shared: Store = Store()
+    
+    var movieReviews: [MovieReview]
+    
+    private init(){
+        movieReviews = [MovieReview]()
```

Comment:
Since `movieReviews` is already a `[MovieReview]`, you only need `movieReviews = []`. You could also set the initial value directly on the property. Finally, if you do want to set it in the initializer, taking the array as a parameter which defaults to an empty array could help testing in the future.

Reply:
As you saw in our last meeting, I am no longer setting this in an initializer.